ansible/example_ixia

```diff
@@ -0,0 +1,22 @@
+[sonic_dell64_40]
+example-s6100-dut-1     ansible_host=10.0.0.10
+
+[sonic_dell64_40:vars]
+hwsku="Force10-S6100"
+iface_speed='40000'
+
+[ixia_chassis]
+example-ixia-1          ansible_host=10.0.0.20   os=ixia
+
+[sonic:children]
+sonic_dell64_40
+ixia_chassis
```

Comment:
ixia_chassis should not be classified as a sonic child

this file should have another definition
[example_ixia:children]
sonic
ixia_chassis

Reply:
Thanks. Moved.